docs/data_repository_template.md

```diff
@@ -0,0 +1,81 @@
+
+## 📚 Contents
+
+- :ref:`data-repo-structure`
+- :ref:`data-config-file`
+- :ref:`documentation`
+- :ref:`data-license`
+- :ref:`example-data-repos`
+
+.. _data-repo-structure:
+
+## 🧱 Data repo structure
+
+We recommend to set up a data repo for scivision with the following structure, with the default name for the config file that the scivision API will recognise being `.scivision/data.yml`:
+
+```
+exampleuser/data_repo
+│   README           
+│   LICENSE          
+│   
+└───.scivision
+│   │   data.yml     
+```
+
+.. _data-config-file:
+
+## 🖋️ Data config file
+
+The `scivision` python API relies upon another open source python tool called `intake` for loading datasets. We recommend creating a YAML format "intake catalog" (config file), to enable scivision users who discover your dataset to easily load it via the API.
+
+For a comprehensive guide to setting up this data config file (intake catalog), consult the [intake documentation](https://intake.readthedocs.io/en/latest/catalog.html#yaml-format).
+
+Here is an example of a `.scivision/data.yml` config (intake catalog) taken from the GitHub repo [alan-turing-institute/plankton-cefas-scivision](https://github.com/alan-turing-institute/plankton-cefas-scivision):
+
+```yaml
+sources:
+  plankton:
+      description: Sample images required to demonstrate the ResNet50 model trained in the Rapid Identification of Plankton using Machine Learning DSG undertaken by Cefas, The Alan Turing Institute and Plankton Analytics Ltd. 
+      origin: 
+      driver: intake_xarray.image.ImageSource
+      args:
+        urlpath: ["zip://*.tif::https://zenodo.org/record/6143685/files/images.zip"]
+        chunks: {}
+        storage_options: {'anon': True}
+        coerce_shape: [1000, 1000]
+        exif_tags: True
+```
+
+In order to fully understand how this `data.yml` has been configured, you should consult the [intake documentation](https://intake.readthedocs.io/en/latest/catalog.html#yaml-format). Here we describe just the key fields:
+
+- `sources`: gives the option to include multiple datasets in a single config, in this case there is just one called "plankton"
+- `driver`: the intake "driver" plugin, which specifies the format that the data is loaded to, in this case [xarray](https://github.com/intake/intake-xarray)
+- `urlpath`: a path to the data itself; in this example, we point to a Zenodo repository and load all the `.tif` files from the compressed zip file stored there
+
+
+The scivision API can accept a path to a locally stored data config file, or the GitHub repository containing it as specified in this guide. See the `load_dataset` function in the :ref:`api-docs` docs.
+
+.. _documentation:
+
+## 📄 Documentation
+
+A `README`, which includes helpful information on the dataset and its origin should be included. Without this, your dataset may not be accepted for inclusion in the scivision catalog.
+
+.. _data-license:
+
+## 📜 Data licence
```

Comment:
```suggestion
## 📜 Data license
```
Note the text below you refer license and not licence.

Reply:
Ok well controversial to go with the US spelling however I agree because we also use "Catalog" instead of "Catalogue" 😃